src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -72,7 +72,8 @@ Copyright (c) .NET Foundation. All rights reserved.
                    '$(PublishAot)' == 'true'
                    )
                  )">
-    <RuntimeIdentifier>$(NETCoreSdkPortableRuntimeIdentifier)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="$(NETCoreSdkRuntimeIdentifier.StartsWith('linux-musl'))">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkPortableRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
We should instead fix `NETCoreSdkPortableRuntimeIdentifier` so it is `linux-musl-{arch}` on Alpine.

Reply:
It's defined in the installer repo, correct?

Reply:
Yes. And it looks like the `x64` build gets it right:
```
    <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion>
    <BundledNETCoreAppTargetFrameworkVersion>6.0</BundledNETCoreAppTargetFrameworkVersion>
    <BundledNETCoreAppPackageVersion>6.0.9</BundledNETCoreAppPackageVersion>
    <BundledNETStandardTargetFrameworkVersion>2.1</BundledNETStandardTargetFrameworkVersion>
    <BundledNETStandardPackageVersion>2.1.0</BundledNETStandardPackageVersion>
    <BundledNETCorePlatformsPackageVersion>6.0.5</BundledNETCorePlatformsPackageVersion>
    <BundledRuntimeIdentifierGraphFile>$(MSBuildThisFileDirectory)RuntimeIdentifierGraph.json</BundledRuntimeIdentifierGraphFile>
    <NETCoreSdkVersion>6.0.401</NETCoreSdkVersion>
    <NETCoreSdkRuntimeIdentifier>linux-musl-x64</NETCoreSdkRuntimeIdentifier>
    <NETCoreSdkPortableRuntimeIdentifier>linux-musl-x64</NETCoreSdkPortableRuntimeIdentifier>
```

Reply:
Unlike the `arm64` build:
```
    <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion>
    <BundledNETCoreAppTargetFrameworkVersion>6.0</BundledNETCoreAppTargetFrameworkVersion>
    <BundledNETCoreAppPackageVersion>6.0.9</BundledNETCoreAppPackageVersion>
    <BundledNETStandardTargetFrameworkVersion>2.1</BundledNETStandardTargetFrameworkVersion>
    <BundledNETStandardPackageVersion>2.1.0</BundledNETStandardPackageVersion>
    <BundledNETCorePlatformsPackageVersion>6.0.5</BundledNETCorePlatformsPackageVersion>
    <BundledRuntimeIdentifierGraphFile>$(MSBuildThisFileDirectory)RuntimeIdentifierGraph.json</BundledRuntimeIdentifierGraphFile>
    <NETCoreSdkVersion>6.0.401</NETCoreSdkVersion>
    <NETCoreSdkRuntimeIdentifier>linux-musl-arm64</NETCoreSdkRuntimeIdentifier>
    <NETCoreSdkPortableRuntimeIdentifier>linux-arm64</NETCoreSdkPortableRuntimeIdentifier>
```

Reply:
Yup, I found the same behavior. Do you know of quick way to test this installer patch:

```diff
--- a/src/redist/targets/GetRuntimeInformation.targets
+++ b/src/redist/targets/GetRuntimeInformation.targets
@@ -26,7 +26,7 @@
     <PropertyGroup>
       <ProductMonikerRid Condition=" '$(Rid)' == 'ubuntu.16.04-x64' OR
                                    '$(Rid)' == 'rhel.6-x64' OR
-                                   '$(Rid)' == 'linux-musl-x64' ">$(Rid)</ProductMonikerRid>
+                                   $(Rid.StartsWith('linux-musl') ">$(Rid)</ProductMonikerRid>
       <ProductMonikerRid Condition=" '$(ProductMonikerRid)' == '' ">$(OSName)-$(Architecture)</ProductMonikerRid>
 
       <PortableProductMonikerRid Condition=" '$(PortableProductMonikerRid)' == '' ">$(HostOSName)-$(Architecture)</PortableProductMonikerRid>
```